.travis.yml

```diff
@@ -1,6 +1,6 @@
 sudo: false
 language: java
-jdk: oraclejdk7
+jdk: oraclejdk8
```

Comment:
Why Java 8? Is that required now for Android dev and runtime? Fine by me, just want to confirm.

Reply:
It is not about using Java 8. Travis-CI is no longer supporting oraclejdk7. Now, they only support oraclejdk8, oraclejdk9 or openjdk7.

please see,
https://github.com/travis-ci/travis-ci/issues/7884#issuecomment-308451879
https://github.com/travis-ci/travis-ci/issues/7019#issuecomment-316769716

Reply:
To be honest I would rather run the build with openjdk7 ... wanna change to that and lets see how that goes? Alternatively I can switch in master and we can see there and have that as a separate commit/change from this PR.

Reply:
It seems, the project compiles well with OpenJDK7, too.